en/config_vtol/vtol_tailsitter_tuning.md

```diff
@@ -0,0 +1,66 @@
+- Pitch rate control ([MC_PITCHRATE_P](../advanced_config/parameter_reference.md#MC_PITCHRATE_P) = 0, [MC_PITCHRATE_I](../advanced_config/parameter_reference.md#MC_PITCHRATE_I) = 0, [MC_PITCHRATE_D](../advanced_config/parameter_reference.md#MC_PITCHRATE_D) = 0, [MC_PITCHRATE_K](../advanced_config/parameter_reference.md#MC_PITCHRATE_K) = 1)
+- Yaw rate control ([MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P) = 0, [MC_YAWRATE_I](../advanced_config/parameter_reference.md#MC_YAWRATE_I) = 0, [MC_YAWRATE_D](../advanced_config/parameter_reference.md#MC_YAWRATE_D) = 0, [MC_YAWRATE_K](../advanced_config/parameter_reference.md#MC_YAWRATE_K) = 1)
+
+- Angle control ([MC_ROLL_P](../advanced_config/parameter_reference.md#MC_ROLL_P) = 0, ([MC_PITCH_P](../advanced_config/parameter_reference.md#MC_PITCH_P) = 0, ([MC_YAW_P](../advanced_config/parameter_reference.md#MC_YAW_P) = 0)
+
+This is just a sanity check, and the expected behaviour is that when you arm in Stabilized mode, both the motors should spin, control surfaces should be centred, and no effect of manual RC input using the transmistter is observed. Moreover, if possible, gently pick up the vehicle by hand and try tilting it in different axes. YOU SHOULD FEEL NO COUNTERING TORQUE EXERTED BY THE MOTORS OR THE CONTROL SURFACES. 
+
+Once this sanity check is complete, we move on to increasting the gains slowly. 
+
+### Rate Gains
+
+- Increase [MC_ROLLRATE_P](../advanced_config/parameter_reference.md#MC_ROLLRATE_P) by 0.1, and try arming and lifting the vehicle and displacing in the roll axis. You should feel a countering torque and the quad trying to reorient itself. This is still a very manual process, and there is not much science to it. Increase the proportional gain until you can feel sustained oscillations when you displace it. Then decrease it to 1/2 of that value. If your osciallations never converge, it is worth trying to increase the [MC_ROLLRATE_I](../advanced_config/parameter_reference.md#MC_ROLLRATE_I) gain to a very small value. If osciallations are the problem, try increasing [MC_ROLLRATE_D](../advanced_config/parameter_reference.md#MC_ROLLRATE_D)
+
+- Do the same as above for Pitch and Yaw. 
+
+
+### Angle Gains
+
+Once you have a tuned rate controller, your vehicle can hover in a very stable way (albeit with some support for position control, since your angle gains are still 0 and your RC transmitter will not work).
+
+You should then move on to tuning the Angle gains. In most cases, just a Proportional controller is sufficient enough for this. 
+
+- Repeat the same process as mentioned above in the Rate controller gain tuning, but instead of manually displacing the vehicle in a particular axes, use the RC transmitter to give pulses in certain axes and check the responce. Tune the Proportional gains so that there are no oscillations, and if they do exist, ensure that they subside quickly.
+
+
+## Duo Tailsitter FW Mode Tuning Guide
```

Comment:
Do you fly with or without an airspeed sensor? As that changes how you should approach the tuning. With an airspeed sensor the airspeed scaling should take care of the gain scheduling automatically, while if you fly without airspeed sensor you need to tune it through the setting of FW_AIRSPD_STALL and FW_AIRSPD_TRIM. The bigger those two numbers are apart, the more gain scheduling happens.